tox.ini

```diff
@@ -28,6 +28,7 @@ commands =
 
 [testenv:dev-api]
 envdir = .tox/py37-linux/
+passenv = PAASTA_TEST_CLUSTER KUBECONFIG PAASTA_SYSTEM_CONFIG_DIR
```

Comment:
Note I added this in as without it, I needed to actually export these in my environment, whereas the wiki instructs you to just declare them when you run `make dev-api` e.g. `PAASTA_SYSTEM_CONFIG_DIR=etc_paasta_for_deployment make dev-api`.